src/components/BackButton/StyledBackButton.tsx

```diff
@@ -5,6 +5,8 @@ import colors from '@ui/styles/colors';
 
 export const StyledBackButton = styled(Button)`
   border-radius: 10px;
+  margin-left: 3em;
```

Comment:
You should move these margin properties up into a parent layout, see my above comment.

```suggestion
```